packages/edge/test/edge-headers.test.ts

```diff
@@ -15,6 +16,11 @@ import {
   REQUEST_ID_HEADER_NAME,
 } from '../src';
 
+test('vitest environment is correctly added', () => {
+  // @ts-ignore
+  expect(globalThis.EdgeRuntime).toBeDefined();
```

Comment:
I wasn't really sure how to validate that `@vitest-environment edge-runtime` actually did anything, this works but might be nice to have a more meaningful test.

Having said that, the other tests all pass without the `@vitest-environment edge-runtime` header so we may want to test something more specific.